edx_solutions_api_integration/courses/serializers.py

```diff
@@ -46,7 +47,7 @@ def get_completion_percentage(self, obj):
         formats get completions as percentage
         """
         total_completions = self.context['total_completions'] or 0
```

Comment:
Unless I misunderstand the purpose here, total completions is part of the api now.  It can just be taken from `obj['possible']`